content-src/components/A11yLinkButton/_A11yLinkButton.scss

```diff
@@ -0,0 +1,11 @@
+
+.a11y-link-button {
+  border: 0;
+  padding: unset;
```

Comment:
I don't think you get anything extra from `unset` here, `0` would be more appropriate